sapphire/transformations/celestial.py

```diff
@@ -168,18 +184,22 @@ def equatorial_to_horizontal(latitude, longitude, timestamp, right_ascension,
 
     zenith, azimuth = horizontal_to_zenithazimuth(altitude, alt_azimuth)
 
-    return zenith, azimuth
+    return zenith, azimuth # NB: return is in zenithazimuth format not horizontal
```

Comment:
Okay-I can fix the function name but I would rather throw it out entirely in that case. Either way, it will break code for people. Is removing it okay? The newer astropy versions supersede it anyway.